README.md

```diff
@@ -5,11 +5,33 @@ A digital repository application in use at Princeton University Library for stor
 [![CircleCI](https://circleci.com/gh/pulibrary/figgy.svg?style=svg)](https://circleci.com/gh/pulibrary/figgy)
 [![Browserstack](./browserstack-logo.svg)](https://www.browserstack.com/)
 
-## Dependencies
+## Language Dependencies
+
+Figgy provides a `.tool-versions` file -- consule this file for required languages and their current versions. `asdf` users should ensure all plugins listed there are installed:
+
+`$ asdf plugin-add ruby`
+`$ asdf plugin-add nodejs`
+`$ asdf plugin-add java`
```

Comment:
@hackartisan I was in main when I did asdf install. So it installed what was in the .tools-version. I think you can ignore my previous comment. I had to ran asdf plugin-add java because I didnt have it at the moment in the .tools-version.

Reply:
Interesting. I do think you have to add them manually, but I could be wrong. Maybe @zaymasood or @XanderGardner can try it without adding them when they're setting up their new machines.